app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -377,6 +377,11 @@ public void onCreate(Bundle savedInstanceState) {
 
     initPlayStoreUri();
 
+    if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP){
+    } else{
```

Comment:
Formatting 

if () {

} else {

}

use static imports for the SDK_INT stuff if possible.

Reply:
@mhutti1 Should this be fine?

import static android.os.Build.VERSION.SDK_INT;
import static android.os.Build.VERSION_CODES;
 . 
 .
 .
 if (SDK_INT > VERSION_CODES.LOLLIPOP){
      
    } else{
      snackbarLayout.setFitsSystemWindows(true);
    }